x-pack/platform/test/api_integration/services/data_view_api.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export function DataViewApiProvider({ getService }: FtrProviderContext) {
```

Comment:
I see this is also defined under `x-pack/test/api_integration/deployment_agnostic/services/data_view_api.ts`.
I assume the goal is to relocate the whole deployment_agnostic folder as well (in follow up tasks?) and remove duplicates

Reply:
"DA" services are unique by design, more restricted and generally may not work in non-DA tests:
- SAML authentication only, but `FtrProviderContext` used by default in api tests allows both basic and saml.
- system indexes / operator user is not available, but `FtrProviderContext` exposes `supertest` service authenticated with highest privileges.

Your point is valid, but I'm afraid we have to keep "DA" services separately even if it means name duplication. We discussed with @pheyos it and conclusion was that SKA test relocation focuses around split and we don't refactor/unify services to not slow process.